tutorials/ai-core-template-wizard/ai-core-template-wizard.md

```diff
@@ -0,0 +1,74 @@
+---
+parser: v2
+auto_validation: true
+time: 45
+tags: [ tutorial>beginner, topic>artificial-intelligence, topic>machine-learning, software-product>sap-business-technology-platform, software-product>sap-ai-launchpad, software-product>sap-ai-core ]
+primary_tag: software-product>sap-ai-core
+author_name: Dhrubajyoti Paul
+author_profile: https://github.com/dhrubpaul
+---
+
+# Deploying a movie recommendation system to AI core using Tempelate generator
```

Comment:
Please use Title Case for the tutorial title.
"Deploy a Movie Recommendation System to AI Core Using Template Generator"